src/ds/ds6/ds6-private.h

```diff
@@ -0,0 +1,7 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
```

Comment:
As discussed if we add it, lets populate it..

Reply:
done